packages/manager/modules/pci/src/projects/new/new.html

```diff
@@ -38,29 +39,24 @@ <h1 data-translate="pci_projects_new_title">
                     </div>
 
                     <div class="d-flex flex-row-reverse justify-content-between align-items-center my-3">
-                        <oui-button
-                            data-ng-if="$ctrl.isNextButtonVisible()"
-                            data-variant="primary"
-                            data-type="submit"
-                            data-disabled="$ctrl.isNextButtonDisabled()"
-                            data-variant-nav="next"
-                            data-on-click="$ctrl.onNextBtnClick()">
-                            <span data-ng-bind="$ctrl.getNextButtonText()"></span>
-                        </oui-button>
-
-                        <a class="oui-button oui-button_primary oui-button_icon-right"
-                        data-ng-href="{{ $ctrl.getNextLinkHref() }}"
-                        data-ng-if="!$ctrl.isNextButtonVisible()">
+                        <button class="oui-button oui-button_primary oui-button_icon-right"
+                                type="submit" form="projectDescriptionForm"
+                                data-ng-disabled="$ctrl.isNextButtonDisabled()"
+                                data-ng-click="$ctrl.onNextBtnClick()">
                             <span data-ng-bind="$ctrl.getNextButtonText()"></span>
                             <span class="oui-icon oui-icon_circle oui-icon-chevron-right"
-                                aria-hidden="true">
+                                  aria-hidden="true">
                             </span>
-                        </a>
+                        </button>
 
                         <a data-ng-href="{{ $ctrl.getStateLink('cancel') }}"
-                        data-translate="pci_projects_new_cancel">
+                           data-translate="pci_projects_new_cancel">
                         </a>
                     </div>
+
+                    <p data-ng-if="$ctrl.isCreditAsteriskVisible()">
+                        <small data-translate="pci_projects_new_create_credit_asterisk"></small>
```

Comment:
`small` should be used only for side-comments and small prints (copyright notices and legal texts)